debezium-event-api/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```

Comment:
@rhauch are you sure about a new Maven module ? Couldn't this code be part of `embedded` ?